sim/battle.ts

```diff
@@ -2219,7 +2219,7 @@ export class Battle {
 			}
 		}
 
-		if (this.checkWin(faintData)) return true;
+		if (checkWin && this.checkWin(faintData)) return true;
 
 		if (faintData && length) {
 			this.runEvent('AfterFaint', faintData.target, faintData.source, faintData.effect, length);
```

Comment:
How _should_ `checkWin` interact with `AfterFaint`?

Reply:
If the battle's over these effects don't happen.